java/code/src/com/redhat/rhn/domain/reactor/SaltEvent.java

```diff
@@ -70,7 +70,7 @@ public String getMinionId() {
      * @param minionIdIn the new id
      */
     public void setMinionId(String minionIdIn) {
-        this.minionId = minionId;
+        this.minionId = minionIdIn;
```

Comment:
Ouch, this was pretty bad. I would guess this setter was never used, otherwise it would have been a problem. 

I would evaluate if it's possible to remove all the setters and turn this pojo into an immutable object.